### ISSUE TITLE
Regression: Ctrl+Backspace or Opt+Del do not properly delete entire words

### ISSUE THREAD

**Issue by ClairelyClaire** [WordPress/gutenberg#11556]
**Describe the bug**
In Windows, Ctrl+Backspace allows deleting entire words, rather than individual characters. Gutenberg no longer respects this since the last update.

**To Reproduce**
1. Create a post or page.
2. Type a sentence.
3. Hold **ctrl** and press **backspace**.
4. Single character is deleted.

**Expected behavior**
Whole words should be deleted with this keyboard command.
_Edit by mcsf:_ Expectation coded here: https://github.com/WordPress/gutenberg/issues/11556#issuecomment-437081127

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Edge
 - Version: 44.18277.1002.0

**Additional context**
Gutenberg v4.2.0

**Comment by estherwn** [comment]
I have the same issue after updating Gutenberg today to version 4.2.0 using Firefox v63.0.1

**Comment by designsimply** [comment]
Tested with WordPress 4.9.8 and Gutenberg 4.2.0 and confirmed that Control-Backspace (Windows) and Option-Delete (Mac) does not delete entire words.

<sup>Tested with Edge 17 on Windows 10 and Firefox 63.0.1 on macOS 10.13.6.</sup>

**Comment by mcsf** [comment]
I can confirm the issue. Additionally, on Mac, <kbd>cmd+del</kbd>, expected to wipe the current line, only deletes the last character.

**Comment by mcsf** [comment]
Any fix for this should add the following e2e test:

```diff
diff --git a/test/e2e/specs/__snapshots__/rich-text.test.js.snap b/test/e2e/specs/__snapshots__/rich-text.test.js.snap
index fc96d2d6f..4dad94c54 100644
--- a/test/e2e/specs/__snapshots__/rich-text.test.js.snap
+++ b/test/e2e/specs/__snapshots__/rich-text.test.js.snap
@@ -18,6 +18,12 @@ exports[`RichText should apply formatting with primary shortcut 1`] = `
 <!-- /wp:paragraph -->"
 `;
 
+exports[`RichText should delete whole words using native keyboard shortcuts 1`] = `
+"<!-- wp:paragraph -->
+<p>Some text</p>
+<!-- /wp:paragraph -->"
+`;
+
 exports[`RichText should handle change in tag name gracefully 1`] = `
 "<!-- wp:heading {\\"level\\":3} -->
 <h3></h3>
diff --git a/test/e2e/specs/rich-text.test.js b/test/e2e/specs/rich-text.test.js
index 9203ec32c..09cb73033 100644
--- a/test/e2e/specs/rich-text.test.js
+++ b/test/e2e/specs/rich-text.test.js
@@ -58,4 +58,12 @@ describe( 'RichText', () => {
 
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 	} );
+
+	it( 'should delete whole words using native keyboard shortcuts', async () => {
+		await clickBlockAppender();
+		await page.keyboard.type( 'Some text here' );
+		await pressWithModifier( META_KEY, 'Backspace' );
+
+		expect( await getEditedPostContent() ).toMatchSnapshot();
+	} );
 } );
```

**Comment by mcsf** [comment]
Closed by #11627.
